api/server/server.ts

```diff
@@ -104,12 +105,29 @@ type ErrorOpenApi =
   | Forbidden;
 
 app.use(
-  (err: ErrorOpenApi, req: Request, res: Response, next: NextFunction) => {
-    // format error
-    res.status(err.status || 500).json({
-      message: err.message,
-      errors: err.errors,
-    });
+  (
+    err: Error | ErrorOpenApi,
+    req: Request,
+    res: Response,
+    next: NextFunction,
+  ) => {
+    if (
+      (err as ErrorOpenApi).message &&
+      (err as ErrorOpenApi).errors &&
+      (err as ErrorOpenApi).status
+    ) {
+      // The error is throw by OpenApiValidator
+      Logger.info(err);
+
+      res.status((err as ErrorOpenApi).status || 500).json({
+        msg: (err as ErrorOpenApi).message,
```

Comment:
Can you keep variable as message and not msg as we use it in front

Reply:
Done